test/blackbox-tests/test-cases/install-glob/install-glob-files-only.t

```diff
@@ -0,0 +1,33 @@
+Test that the glob_files terms are only accepted in the files field and not in
```

Comment:
```suggestion
Test that the ``glob_files`` terms are only accepted in the ``files`` field and not in
```

Reply:
This syntax is only available in .rst files